.github/workflows/check_pr.yml

```diff
@@ -49,16 +49,16 @@ jobs:
       matrix:
         os: [macos-latest, windows-latest, ubuntu-latest]
         # node versions to run tests on
-        nodeRun: [16]
+        nodeRun: [18]
         # node versions on which code was built and should be tested
-        nodeBuild: [16]
+        nodeBuild: [18]
         include:
           - os: ubuntu-latest
-            nodeRun: 14
-            nodeBuild: 16
+            nodeRun: 16
+            nodeBuild: 18
           - os: ubuntu-latest
-            nodeRun: 18
-            nodeBuild: 16
+            nodeRun: 16
+            nodeBuild: 18
 
```

Comment:
Not sure about this one. The first ubuntu-latest has to be nodeRun 16 & nodeBuild 18. But whatsfor the second one?

Reply:
Like this?
```typescript
 include:
          - os: ubuntu-latest
            nodeRun: 16
            nodeBuild: 18
          - os: ubuntu-latest
            nodeRun: 18
            nodeBuild: 18
```

Reply:
Missed it. We don't need to include 18/18 again, because that's coming directly from the initial definition